src/domTree.js

```diff
@@ -183,6 +183,12 @@ function symbolNode(value, height, depth, italic, skew, classes, style) {
     this.classes = classes || [];
     this.style = style || {};
     this.maxFontSize = 0;
+
+    if (cjkRegex.test(value)) {
+        // Force serif fonts for CJK glyphs because some browsers/OSs default
+        // to a sans-serif font for these glyphs.
+        this.style['font-family'] = 'serif';
+    }
```

Comment:
Can you do this in a css class, instead of using the style attribute? A class can be overridden by the site admin by adding some more css, but a style can not (except perhaps by abusing the important flag). Or perhaps multiple classes, one for each script?